src/uniparc/config/UniParcXRefsColumnConfiguration.tsx

```diff
@@ -0,0 +1,322 @@
+import { ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+import { ExternalLink } from 'franklin-sites';
+
+import Timeline from '../components/entry/Timeline';
+import {
+  EntryType,
+  EntryTypeIcon,
+} from '../../shared/components/entry/EntryTypeIcon';
+import TaxonomyView from '../../shared/components/entry/TaxonomyView';
+
+import { getEntryPath } from '../../app/config/urls';
+
+import parseDate from '../../shared/utils/parseDate';
+
+import { Namespace } from '../../shared/types/namespaces';
+import { ColumnConfiguration } from '../../shared/types/columnConfiguration';
```

Comment:
Type imports want to happen last according to our conventions

Reply:
Good catch, I let that one though, should be fixed now